examples/react-native/storybook/stories/PasswordField.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { storiesOf } from '@storybook/react-native';
+
+import { PasswordField } from '@aws-amplify/ui-react-native/dist/primitives';
+
+const styles = StyleSheet.create({
+  container: {
+    width: '50%',
+  },
+});
+
+storiesOf('PasswordField', module)
+  .add('default', () => <PasswordField containerStyle={styles.container} />)
+  .add('disabled', () => (
+    <PasswordField containerStyle={styles.container} disabled />
+  ));
```

Comment:
Can we include stories for the `hideShowPassword` and `iconStyle` props too?

Reply:
Done!